src/ResetPasswordHelper.php

```diff
@@ -154,6 +154,20 @@ public function getTokenLifetime(): int
         return $this->resetRequestLifetime;
     }
 
+    /**
+     * Generate a fake reset token.
```

Comment:
Clarify usage (so add this):

> Use this to generate a fake token so that you can, for example, show a "reset confirmation email sent"
> page that includes a valid "expiration date", even if the email was not actually found (and so, a true
> ResetPasswordToken was not actually created).

Reply:
done